social/history/v0.1.0/schema.json

```diff
@@ -0,0 +1,67 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://common.schemas.verida.io/social/history/v0.1.0/schema.json",
+    "title": "History",
+    "titlePlural": "Histories",
+    "description": "General history of activities such as listening, watching, browsing, etc.",
+    "type": "object",
+    "appearance": {
+        "style": {
+            "color": "#4A90E2",
+            "icon": "./icon.svg"
+        }
+    },
+    "database": {
+        "name": "social_history",
+        "indexes": {
+            "activityType": [
+                "activityType"
+            ],
+            "source": ["sourceApplication", "sourceId"]      
+        }
+    },
+    "layouts": {
+        "create": [
+            "timestamp",
+            "activityType"
+        ],
+        "view": [
+            "timestamp",
+            "activityType",
+            "name",
+            "summary",
+            "url",
+            "sourceApplication",
+            "sourceId"            
+        ]
+    },
+    "allOf": [
+        {
+            "$ref": "https://core.schemas.verida.io/base/v0.1.0/schema.json"
+        },
+        {
+            "properties": {
```

Comment:
Add `duration` as an optional field (measured in seconds)

Reply:
Added